parm/config/config.defaults.s2sw

```diff
@@ -12,20 +12,17 @@ FHOUT_GFS=6
 FHOUT_HF_GFS=-1
 
 # config.fcst
-IALB=2
-IEMS=2
-DO_CA="YES"
-min_seaice="1e-6"
+min_seaice="1.0e-6"
 use_cice_alb=".true."
-export iopt_sfc="3"
 
 # config.fv3
 DELTIM=300
-layout_x_gfs=8
-layout_y_gfs=12
+layout_x_gfs=24
+layout_y_gfs=16
```

Comment:
Remind me, is S2SW only run in free-forecast mode right now or both ff and cycled? This would impact cycled enkf C384 if both.

Reply:
Only free-forecast right now.